packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -64,21 +74,21 @@ export function computeValue(
       formattedSeries,
     );
 
-    const range = axis.reverse
-      ? [drawingArea.left + drawingArea.width, drawingArea.left]
-      : [drawingArea.left, drawingArea.left + drawingArea.width];
+    const range = getRange(drawingArea, axisName, axis.reverse);
 
     if (isBandScaleConfig(axis)) {
       const DEFAULT_CATEGORY_GAP_RATIO = 0.2;
       const DEFAULT_BAR_GAP_RATIO = 0.1;
 
       const categoryGapRatio = axis.categoryGapRatio ?? DEFAULT_CATEGORY_GAP_RATIO;
-      const barGapRatio = axis.barGapRatio ?? DEFAULT_BAR_GAP_RATIO;
+      const barGapRatio = axisName === 'x' ? axis.barGapRatio ?? DEFAULT_BAR_GAP_RATIO : 0;
+      const scaleRange = axisName === 'x' ? range : [...range].reverse();
```

Comment:
```suggestion
      // Reverse range because ordinal scales are presented from top to bottom on y-axis
      const scaleRange = axisName === 'x' ? range : [range[1], range[0]];
```

Reply:
Just to add a bit of context in the codebase